brianlong/www.validators.app
# 183984276 - add npm font awesome free

#### What's this PR do?
PR dops the fontawesome kit and uses the npm package.

#### How should this be manually tested?
The only way is to manually check the web application and verify font awesome occurrences if presented.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183984276)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
